lms/templates/admin/organization.html.jinja2

```diff
@@ -48,12 +48,7 @@
         </div>
 
         {% if org.application_instances %}
-            {{ macros.object_list_table(
-                    request, 'admin.instance', org.application_instances,
-                    ['Consumer key', 'Deployment ID', 'Tool name', 'URL', "Email"],
-                    ['consumer_key', 'deployment_id', 'tool_consumer_instance_name', 'lms_url', 'requesters_email']
-               )
-            }}
+            {{ macros.instances_table(request, org.application_instances) }}
```

Comment:
I don't think these were different on purpose?

Reply:
I reckon they were different on purpose, not sure if the selection of columns made much sense.

The idea was that while you were in an organization you already have some context about things that are potentially the same across all the AIs in it.

GUID is likely to be the same for example. 

It will probably make more sense to revisit the common columns like you are doing instead of trying to fine-tune them for every view.

Reply:
Oh ok! Well hopefully this is better for now?